apps/admin-ui/src/authentication/AuthenticationSection.tsx

```diff
@@ -40,6 +40,7 @@ import { addTrailingSlash } from "../util";
 import { getAuthorizationHeaders } from "../utils/getAuthorizationHeaders";
 
 import "./authentication-section.css";
+import useLocaleSort, { mapByKey } from "../utils/useLocaleSort";
```

Comment:
Nit: this import should be moved up a bit.
```suggestion
import { getAuthorizationHeaders } from "../utils/getAuthorizationHeaders";
import useLocaleSort, { mapByKey } from "../utils/useLocaleSort";

import "./authentication-section.css";
```